cypress/support/helper.js

```diff
@@ -15,6 +15,7 @@ export function visitComponentUrl(component, suffix) {
   cy.visit(prepareUrl(component, suffix), {
     onBeforeLoad,
   });
+  cy.get("#root :first-child", { timeout: 10000 }).should("exist");
```

Comment:
I don't think we need one more `timeout` here. Because we already have it in line 54.